configure.ac

```diff
@@ -192,6 +192,8 @@
     AC_CHECK_FUNCS([strlcpy strlcat])
     CFLAGS=$OCFLAGS
 
+    AC_CHECK_FUNCS([madvise posix_fadvise], [AC_DEFINE([HAVE_ADVISE], [1], [Define if system is willing to listen])])
```

Comment:
This error:
<pre>
configure:14571: checking for posix_fadvise
configure:14571: clang -o conftest -g -O2   conftest.c  >&5
Undefined symbols for architecture x86_64:
  "_posix_fadvise", referenced from:
      _main in conftest-e4c674.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
configure:14571: $? = 1
configure: failed program was:
</pre>
still results in config.h having:
<pre>
/* Define if system is willing to listen */
#define HAVE_ADVISE 1
</pre>